server/api.ts

```diff
@@ -1104,6 +1105,33 @@ router.postAsync("/stage-stats", ensure.isAdmin, async (req, res) => {
   res.send(updatedStageStats);
 });
 
+/**
+ * POST /api/refetch-stats
+ * Refretches the stats of a batch of mp links in a stage
+ * Params:
+ *   - tourney: identifier for the tourney
+ *   - stage: identifier for the stage
+ * Returns:
+ *   - stats: the updated stage stats
+ */
+router.postAsync("/refetch-stats", ensure.isAdmin, async (req, res) => {
+  logger.info(`${req.user.username} initiated a refresh of ${req.body.tourney}/${req.body.stage} stats`);
+  let mpLinks: string[] = [];
+  if (req.body.stage === "Qualifiers") mpLinks = (await QualifiersLobby.find({ tourney: req.query.tourney })).map(lobby => lobby.link);
```

Comment:
i feel like cramming it into one line just makes it harder to read than if it were just:
```
if (req.body.stage === "Qualifiers") {
   const lobbies = await QualifiersLobby.find({ tourney: req.query.tourney });
   mpLinks = lobbies.map(lobby => lobby.link);
} else {
  const match = await Match.find({ tourney: req.body.tourney, stage: req.body.stage });
  mpLinks = match.map(match => match.link);
}
```

Reply:
:^)